configs/default/MTKS-MATEKF405TE.config

```diff
@@ -116,34 +112,38 @@ dma pin A02 0
 feature -RX_PARALLEL_PWM
 feature RX_SERIAL
 feature LED_STRIP
+feature TELEMETRY
 feature OSD
 
 # serial
-serial 20 1 115200 57600 0 115200
-serial 0 0 115200 57600 0 115200
 serial 1 64 115200 57600 0 115200
 serial 2 1024 115200 57600 0 115200
-serial 3 1 115200 57600 0 115200
-serial 4 0 115200 57600 0 115200
-serial 5 0 115200 57600 0 115200
-serial 30 0 115200 57600 0 115200
-
 
 # master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set baro_bustype = I2C
+set baro_i2c_device = 1
+set blackbox_device = SPIFLASH
 set current_meter = ADC
 set battery_meter = ADC
-set dshot_bitbang = ON
-set beeper_frequency = 2500
-set blackbox_device = SPIFLASH
+set vbat_scale = 210
+set ibata_scale = 150
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
 set max7456_spi_bus = 1
-set dashboard_i2c_bus = 0
 set pinio_box = 40,41,255,255
 set flash_spi_bus = 2
-set baro_bustype = I2C
-set baro_i2c_device = 1
-set baro_i2c_address = 0
 set gyro_1_bustype = SPI
 set gyro_1_spibus = 1
 set gyro_1_sensor_align = CW270FLIP
 set gyro_1_align_pitch = 1800
 set gyro_1_align_yaw = 2700
+set vbat_scale = 210
+set serialrx_provider = CRSF
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = AUTO
+set beeper_inversion = ON
+set beeper_od = OFF
```

Comment:
repeated
130->143, 
123,124->145,146
132,133->148,149

Reply:
Grouped variables for clarity and removed duplication.